Directory.Build.props

```diff
@@ -89,7 +89,8 @@
 
     <MicrosoftApplicationInsightsVersion>2.4.0</MicrosoftApplicationInsightsVersion>
     <MicrosoftAzureCosmosTableVersion>1.0.5</MicrosoftAzureCosmosTableVersion>
-    <MicrosoftAzureEventHubsVersion>4.1.0</MicrosoftAzureEventHubsVersion>
+    <AzureMessagingEventHubs>5.1.0</AzureMessagingEventHubs>
+    <AzureIdentityVersion>1.1.1</AzureIdentityVersion>
```

Comment:
Only `Azure.Core` should be added, not Azure.Identity: https://github.com/dotnet/orleans/pull/6648#pullrequestreview-460416498

Reply:
`DefaultAzureCredential` is used in tests.